react/src/components/DefaultProviders.tsx

```diff
@@ -327,6 +327,7 @@ export const DefaultProvidersForReactRoot: React.FC<
                   ? theme.darkAlgorithm
                   : theme.defaultAlgorithm,
               }}
+              csp={{ nonce: '{{nonce}}' }}
```

Comment:
I remember there was a discussion about separating the variable that receives the `{{nonce}}` value and referencing that variable in other locations.